quickstart/kubernetes-guestbook.md

```diff
@@ -0,0 +1,373 @@
+     +   ├─ kubernetes:core:Service     frontend                         create
+     +   ├─ kubernetes:apps:Deployment  redis-master                     create
+     +   ├─ kubernetes:apps:Deployment  redis-slave                      create
+     +   └─ kubernetes:apps:Deployment  frontend                         create
+
+    info: 7 changes previewed:
+        + 7 resources to create
+
+    Do you want to perform this update?
+    > yes
+      no
+      details
+    ```
+
+    Let's select "yes" and hit enter. The deployment will proceed, and the output will look something like this:
+
+    ```
+    Updating stack 'k8s-guestbook-dev'
+    Performing changes:
+
+         Type                           Name                             Status      Info
+     +   pulumi:pulumi:Stack            k8s-guestbook-k8s-guestbook-dev  created
+     +   ├─ kubernetes:core:Service     redis-slave                      created     1 info message
+     +   ├─ kubernetes:core:Service     frontend                         created     1 info message
+     +   ├─ kubernetes:core:Service     redis-master                     created     1 info message
+     +   ├─ kubernetes:apps:Deployment  redis-master                     created
+     +   ├─ kubernetes:apps:Deployment  redis-slave                      created
+     +   └─ kubernetes:apps:Deployment  frontend                         created
+
+    Diagnostics:
+      kubernetes:core:Service: redis-slave
+        info: ✅ Service 'redis-slave' successfully created endpoint objects
+
+      kubernetes:core:Service: frontend
+        info: ✅ Service 'frontend' successfully created endpoint objects
+
+      kubernetes:core:Service: redis-master
+        info: ✅ Service 'redis-master' successfully created endpoint objects
+
+    ---outputs:---
+    frontendIP: "10.102.193.86"
+
+    info: 7 changes performed:
+        + 7 resources created
+    Update duration: 16.226520447s
+
+    Permalink: https://app.pulumi.com/joeduffy/k8s-guestbook-dev/updates/1
+    ```
+
+4.  The application is now running in our Kubernetes cluster.
+
+    Use `kubectl` to see the deployed services:
+
+    ```shell
+    $ kubectl get services
+    ```
+
+    We should see entries for each of the four Services we've created in our program:
+
+    ```
+    NAME           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+    frontend       ClusterIP   10.102.193.86   <none>        80/TCP     2m
+    kubernetes     ClusterIP   10.96.0.1       <none>        443/TCP    1d
+    redis-master   ClusterIP   10.98.205.37    <none>        6379/TCP   2m
+    redis-slave    ClusterIP   10.96.9.70      <none>        6379/TCP   2m
+    ```
+
+    The `pulumi stack output` command prints exported program variables:
+
+    ```shell
+    $ pulumi stack output frontendIP
+    ```
+
+    The value of the `frontendIP` variable matches exactly the `frontend`'s `CLUSTER-IP` reported by `kubectl`:
+
+    ```
+    10.102.193.86
```

Comment:
Should this be part of the code block above?

Reply:
I started explicitly factoring these out. Definitely up for feedback on this. I noticed the Kubernetes tutorials don't put the output and commands together which, I gather, makes it clearer what you're meant to copy and paste, versus just output you should expect. (Coupled with [small touches like copy-to-clipboard icons](https://github.com/pulumi/docs/issues/550) for the commands, this would matter more.)

I don't feel strongly but I do think it flows nicer.